### PR TITLE
Added card component

### PR DESCRIPTION
**Pull request by lghiur**

### CODE REVIEW

File: src/components/Card/sass/Card.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+      margin-right: 10px;
+      grid-row: span 2;
+      width: 42px;
+    }
+  }
+
+  &__title { 
+    font-family: $font-inter-bold;
+    font-size: $lg-font-size;
+    display: box;
```
Comment (ifrim) [inline src/components/Card/sass/Card.scss:41]: I had to look for the `box` value but it doesn't seem to exist. I think it should be `display: flex;` instead.

File: src/components/Card/sass/Card.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    box-orient: vertical;
```
Comment (ifrim) [inline src/components/Card/sass/Card.scss:42]: The same for the `box-orient` property. It doesn't seem to be part of the standard, it looks like it is deprecated. It probably should be `flex-direction: column;` instead.

File: src/components/Card/sass/Card.scss [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+  &__sub-title + &__summary,
+  &__title + &__summary{
+    margin-top: map.get($spacing, 'sm');
+  }
+
+  &__summary {
+    font-family: $font-inter-regular;
+    font-size: $sm-font-size;
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
```
Comment (ifrim) [inline src/components/Card/sass/Card.scss:71]: These seem to be non standard and won't work the same in all the browsers. They should probably be replaced with flex css properties.

**Review by jay-deshmukh — APPROVED** [review]

File: src/components/Card/sass/Card.scss (hunk 1 above)
Comment (lghiur) [reply src/components/Card/sass/Card.scss:41]: https://css-tricks.com/almanac/properties/l/line-clamp/

File: src/components/Card/sass/Card.scss (hunk 2 above)
Comment (lghiur) [reply src/components/Card/sass/Card.scss:42]: https://caniuse.com/?search=display%3A%20-webkit-box

**Review by ifrim — APPROVED** [review]

**Pull request merged**